test/jdk/java/security/KeyStore/PKCS12/java.security

```diff
@@ -0,0 +1,2 @@
+# do not set keystore.type property, so default value will be used
+#keystore.type=
```

Comment:
I think this should not be commented out, otherwise you are still getting the `keystore.type=pkcs12` setting from the `java.security` file in the JDK. To make sure this is working, try temporarily setting this to "foobar" or something random and see if the test fails.

Reply:
Good catch - uncommenting did in fact break the test. To fix, I changed the `@run` flag to be `-Djava.security.properties==${test.src}/java.security CheckDefaults`. By using `==` as opposed to `=` the java.security file is completely overwritten